### PR TITLE
fix(YouTube - Hide Layout components): expandable chips are still shown

### PR DESCRIPTION
**Pull request by johnconner122**
Expandable chips are in exception's path: 
```shell
revanced: LithoFilterPatch: Searching ID: null Path: home_video_with_context.eml|619181af30f6e135|CellType|ContainerType|expandable_metadata.eml|89add36c089fc34|inline_expansion.eml|d8c2397f529050fe|ContainerType|inline_expander.eml|8e73e83e51d8cb1d|ContainerType|ContainerType|ContainerType|ContainerType|styled_product_carousel.eml|80364cdf51436f50|CellType|horizontal_shelf.eml|8dc02154fc601c5f|ContainerType|CollectionType|styled_product_carousel_item.eml|21324e67e3d7194e|CellType|ContainerType|ContainerType|
```

### CODE REVIEW

File: app/src/main/java/app/revanced/integrations/patches/components/LayoutComponentsFilter.java [hunk 1]
```diff
@@ -211,7 +212,8 @@ public LayoutComponentsFilter() {
     public boolean isFiltered(@Nullable String identifier, String path, byte[] protobufBufferArray,
                               FilterGroupList matchedList, FilterGroup matchedGroup, int matchedIndex) {
 
-        if (matchedGroup == notifyMe || matchedGroup == inFeedSurvey) return true;
+        if (matchedGroup == notifyMe || matchedGroup == inFeedSurvey || matchedGroup == expandableMetadata)
```
Comment (oSumAtrIX) [inline app/src/main/java/app/revanced/integrations/patches/components/LayoutComponentsFilter.java:215]: Why is this necessary? Would the last `super.isFiltered` not filter it?

Comment (johnconner122) [reply app/src/main/java/app/revanced/integrations/patches/components/LayoutComponentsFilter.java:215]: Path starts with `home_video_with_context`, which is an exception, so it returns false before it goes to super.isFiltered.
